store/postgres/src/primary.rs

```diff
@@ -78,6 +78,16 @@ table! {
     }
 }
 
+table! {
+    subgraphs.subgraph_features (id) {
+        id -> Text,
+        spec_version -> Text,
+        api_versions -> Array<Text>,
+        features -> Array<Text>,
+        data_sources -> Array<Text>,
+    }
```

Comment:
I like this schema a lot, with the string arrays, a good balance between structure and flexibility.

I'd make `api_version` a nullable `Text`, which should be the `unified_api_version` or `null` if the subgraph is legacy and has mixed versions.

Reply:
done